docs/cmd/kn_plugin_list.md

```diff
@@ -0,0 +1,34 @@
+## kn plugin list
+
+list all visible plugin executables on a user's PATH
+
+### Synopsis
+
+List all available plugin files on a user's PATH.
+
+		Available plugin files are those that are:
+		- executable
+		- anywhere on the user's PATH
```

Comment:
Should mention what we do if we find duplicates - I'm assuming first one wins.

Reply:
Yup, but first one in `PATH` list to be more specific.